src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -81,16 +79,17 @@ const defaultProps = {
         avatarURL: getDefaultAvatar(),
     },
     network: null,
+    currentlyViewedReportID: '',
 };
 
 const SidebarLinks = (props) => {
-    const reportIDInUrl = parseInt(props.match.params.reportID, 10);
+    const activeReportID = parseInt(props.currentlyViewedReportID, 10);
 
     const chatSwitcherStyle = props.isChatSwitcherActive
         ? [styles.sidebarHeader, styles.sidebarHeaderActive]
         : [styles.sidebarHeader];
 
-    const {recentReports} = getSidebarOptions(props.reports, props.personalDetails, props.draftComments, reportIDInUrl);
+    const {recentReports} = getSidebarOptions(props.reports, props.personalDetails, props.draftComments, activeReportID);
```

Comment:
We should fix this. It's a warning but we are treating them as not optional. I have an open issue to make this more clear.

Reply:
Ah, sorry didn't realize what you were referring to at first. You're referring to the fact that this is a linter warning.